File: sumofnnumbers.py
def get(array):

    n=int(input("enter the list size"))
    for w in range(n):
        x=int(input("enter the number"))
        array.append(x)
        w+=1
    return array

def compute(array):

    result=0
    n=len(array)
    result=sum(array)
    return result

def output(result):
    print(result)


def main():
    words=[]
    words=get(words)
"numbers.py" 27 lines, 407 characters

File: sumofnnumbers.c
#include<stdio.h>
void getinput(int *n,int *p);
void compute(int *n,int *p,int *sum);
void output(int *sum);
int main()
{
        int a[10],n,sum=0;
        int *p;
        p=a;
        getinput(&n,p);
        compute(&n,p,&sum);
        output(&sum);
}
void getinput(int *n,int *p)
{
        int i;
        printf("enter the array size\n");

        scanf("%d",n);
        printf("enter the array values\n");
        for(i=0;i<*n;i++)
                scanf("%d",(p+i));
}
void compute(int *n,int *p,int *sum)
{
        int i;
        for(i=0;i<*n;i++)
                *sum=*sum+*(p+i);
}
void output(int *sum)
{
                printf("sum=%d\n",*sum);
}
